mods/tuxemon/l18n/fr_FR/LC_MESSAGES/base.po

```diff
@@ -6292,10 +6292,10 @@ msgid "status_confused"
 msgstr "Confus"
 
 msgid "status_diehard"
-msgstr "Die Hard"
+msgstr "Déterminé"
```

Comment:
Hmmm... I'm not sure if it's the best translation in this case.

Looks like "die hard" can indeed mean "very determined", but I think in this case it's more about "resisting change", or resisting again death itself. Because if I'm not wrong, this status is about "not being able to die".
"Tenace" may be better in my opinion.

Reply:
Oh, really? Well, gonna change that then